client/src/controllers/ActionController.test.js

```diff
@@ -169,4 +169,35 @@ describe('ActionController', () => {
       expect(window.location.assign).not.toHaveBeenCalled();
     });
   });
+
+  describe('select method', () => {
+    beforeEach(() => {
+      document.body.innerHTML = `
+        <textarea 
+          id="text" 
+          rows="1" 
+          data-controller="w-action" 
+          data-action="focus->w-action#select">
+          value="some random text"
+        </textarea>
```

Comment:
```suggestion
          data-action="focus->w-action#select"
        >
          some random text
        </textarea>
```

Just a small typo, test is not impacted but could cause confusion later. the closing `>` is on line 180 so value="..." was actually the whole text, not setting the attribute `value`.

I will fix this up when merging though.